var/spack/repos/builtin/packages/gettext/package.py

```diff
@@ -1,30 +1,15 @@
 from spack import *
 
 class Gettext(Package):
-    """GNU internationalization (i18n) and localization (l10n) library."""
+    """TODO"""
     homepage = "https://www.gnu.org/software/gettext/"
-    url      = "http://ftpmirror.gnu.org/gettext/gettext-0.19.7.tar.xz"
+    url      = "http://ftp.gnu.org/pub/gnu/gettext/gettext-0.19.6.tar.xz"
 
-    version('0.19.7', 'f81e50556da41b44c1d59ac93474dca5')
+    version('0.19.6', '69d79254ee3b41df23f41c2f4fd720d9')
 
-    def install(self, spec, prefix):
-        options = ['--disable-dependency-tracking',
```

Comment:
I am not sure about options, but when i added `gettext` i took them from https://github.com/Homebrew/legacy-homebrew/blob/master/Library/Formula/gettext.rb . Are you sure that they can all be ignored?